### PR TITLE
[MISC] Update dependenices

### PR DESCRIPTION
**Pull request by deusebio**

### CODE REVIEW

File: metadata.yaml [hunk 1]
```diff
@@ -1,15 +1,24 @@
-# Copyright 2023 Canonical Ltd.
+# Copyright 2024 Canonical Ltd.
```
Comment (Batalex) [inline metadata.yaml:1]: **question**: Are we required to keep those up to date?

**Review by Batalex — APPROVED** [review]
LGTM

We need to keep in mind that the export plugin might become mandatory in poetry 2.0

File: metadata.yaml (hunk 1 above)
Comment (zmraul) [reply metadata.yaml:1]: It's usually the year the file was created. For code that ends up being the _Ship of Theseus_ I believe it should be a range? So `# Copyright 2023-2024 Canonical Ltd.` But I don't think we follow any convention yet on our codebases.

**Review by zmraul — APPROVED** [review]

**Comment by deusebio** [comment]
> We need to keep in mind that the export plugin might become mandatory in poetry 2.0

Oh, didn't know. I noticed that it was bringing a lot of dependencies in, that I don't think we need (and that is was creating instabilities). If it becomes mandatory my hope is that they will make it a transitive dependency :crossed_fingers:

File: metadata.yaml (hunk 1 above)
Comment (deusebio) [reply metadata.yaml:1]: We could do this on another PR I suppose. Otherwise the CI will re-run only for this.

**Pull request merged**